runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/DefaultJobBundleFactory.java

```diff
@@ -344,7 +344,7 @@ public void close() throws Exception {
    * now, there is a 1:1 relationship between data services and harness clients. The servers are
    * packaged here to tie server lifetimes to harness client lifetimes.
    */
-  protected static class WrappedSdkHarnessClient implements AutoCloseable {
+  protected static class WrappedSdkHarnessClient {
```

Comment:
Why removing the `AutoCloseable` interface? This still implements `close()`. Did you remove it due to the method not throwing an exception anymore?

Reply:
`AutoCloseable` requires `close() throws Exception` but the new codes neither use try-with-resource with `WrappedSdkHarnessClient` nor throw any exception, so I dropped `AutoCloseable` interface.